src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -14,858 +14,1080 @@ namespace System.Runtime.Intrinsics.X86
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<short> AndNot(Vector128<short> left, Vector128<short> right) => AndNot(left, right);
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<ushort> AndNot(Vector128<ushort> left, Vector128<ushort> right) => AndNot(left, right);
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<int> AndNot(Vector128<int> left, Vector128<int> right) => AndNot(left, right);
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<uint> AndNot(Vector128<uint> left, Vector128<uint> right) => AndNot(left, right);
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<long> AndNot(Vector128<long> left, Vector128<long> right) => AndNot(left, right);
         /// <summary>
-        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_andnot_si128 (__m128i a,  __m128i b);
+        ///   PANDN xmm, xmm/m128
         /// </summary>
         public static Vector128<ulong> AndNot(Vector128<ulong> left, Vector128<ulong> right) => AndNot(left, right);
         /// <summary>
-        /// __m128d _mm_andnot_pd (__m128d a, __m128d b)
+        /// __m128d _mm_andnot_pd (__m128d a, __m128d b);
+        ///   ADDNPD xmm, xmm/m128
         /// </summary>
         public static Vector128<double> AndNot(Vector128<double> left, Vector128<double> right) => AndNot(left, right);
 
         /// <summary>
-        /// __m128i _mm_avg_epu8 (__m128i a,  __m128i b)
+        /// __m128i _mm_avg_epu8 (__m128i a,  __m128i b);
+        ///   PAVGB xmm, xmm/m128
         /// </summary>
         public static Vector128<byte> Average(Vector128<byte> left, Vector128<byte> right) => Average(left, right);
         /// <summary>
-        /// __m128i _mm_avg_epu16 (__m128i a,  __m128i b)
+        /// __m128i _mm_avg_epu16 (__m128i a,  __m128i b);
+        ///   PAVGW xmm, xmm/m128
         /// </summary>
         public static Vector128<ushort> Average(Vector128<ushort> left, Vector128<ushort> right) => Average(left, right);
 
```

Comment:
Seems that `void CasheLineFlush(void* pointer)` or `CLFLUSH` is missing

Reply:
Controlled via the CLFSH CPUID flag, so it would need to be exposed under a separate ISA

Reply:
Right

Reply:
Given that it is a new ISA, we should do it independently.